deployment/ccip/changeset/cs_prerequisites.go

```diff
@@ -245,10 +242,9 @@ func deployPrerequisiteContracts(e deployment.Environment, ab deployment.Address
 				}
 			})
 		if err != nil {
-			lggr.Errorw("Failed to deploy weth9", "err", err)
+			lggr.Errorw("Failed to deploy weth9", "chain", chain.Name, "err", err)
 			return err
 		}
-		lggr.Infow("deployed weth9", "addr", weth.Address)
```

Comment:
yeah I like the positive logs, let add em everywhere

Reply:
Added with deployContract - https://github.com/smartcontractkit/chainlink/blob/CCIP-4506-log-improvements/deployment/helpers.go#L149